<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Junghyun Jung</title> <meta name="author" content="Junghyun Jung"> <meta name="description" content="Department of Computational Biomedicine at Cedars-Sinai Medical Center. "> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/NATIVE.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%87%B0%F0%9F%87%B7&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="http://junghyunjj.github.io//"> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <div class="navbar-brand social"> <a href="mailto:%6A%75%6E%67%68%79%75%6E.%6A%75%6E%67@%63%73%68%73.%6F%72%67" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0001-6832-4368" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=3Va15VcAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/junghyunJJ" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/junghyun-jung-0197b29b" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a> <a href="https://twitter.com/junghyunjj219" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a> </div> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item "> <a class="nav-link" href="/assets/pdf/JJ_CV.pdf">cv</a> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Junghyun</span> Jung </h1> <p class="desc">Research data scientist at Cedars-Sinai Medical Center | Computational biologist</p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/junghyun_jung-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/junghyun_jung-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/junghyun_jung-1400.webp"></source> <img src="/assets/img/junghyun_jung.png" class="img-fluid z-depth-1 rounded" width="auto" height="auto" alt="junghyun_jung.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <div class="address"> <p>Pacific Design Center</p> <p>700 N. San Vicente Blvd.</p> <p>West Hollywood, CA 90069</p> </div> </div> <div class="clearfix"> <p>My name is Junghyun Jung. I’m currently a Research Data Scientist in the <a href="https://www.cedars-sinai.edu/research/departments-institutes/computational-biomedicine.html" rel="external nofollow noopener" target="_blank">Department of Computational Biomedicine</a> at <a href="https://www.cedars-sinai.org/" rel="external nofollow noopener" target="_blank">Cedars-Sinai Medical Center</a>.</p> <p>My research focuses on</p> <ul> <li>establishing agentic AI systems through a multi-agent framework to enhance autonomous decision-making in biological data analysis.</li> <li>developing novel computational tools for the analysis of spatial omics and single-cell eQTL datasets.</li> <li>integrating and analyzing multi-omics data to uncover the molecular mechanisms underlying complex traits and diseases.</li> </ul> <p>I received my Ph.D. in Life Science from <a href="https://www.dongguk.edu/eng/main" rel="external nofollow noopener" target="_blank">Dongguk University</a> in Korea, where I studied multi-omics integrative analysis and method development for functional genomic data. While earning my Ph.D., I was advised by <a href="https://cblab.yolasite.com/" rel="external nofollow noopener" target="_blank">Dr. Jong Wha (Joanne) Joo</a> and <a href="https://sites.google.com/view/dgu-janglab" rel="external nofollow noopener" target="_blank">Dr. Wonhee Jang</a>.</p> <p>I also was a postdoctoral scholar in <a href="https://www.mancusolab.com/" rel="external nofollow noopener" target="_blank">Dr. Nicholas Mancuso’s lab</a> at <a href="https://www.keckmedicine.org/" rel="external nofollow noopener" target="_blank">Keck School of Medicine USC</a> and <a href="https://mangul-lab-usc.github.io/" rel="external nofollow noopener" target="_blank">Dr. Serghei Mangul’s lab</a> at <a href="https://pharmacyschool.usc.edu/" rel="external nofollow noopener" target="_blank">USC School of Pharmacy</a>. My work aimed to perform integrative analysis using Transcriptome-wide association study (TWAS) and analyze GWAS and WES data related to COVID-19.</p> <p>Please feel free to contact me through <a href="mailto:junghyun.jung@cshs.org">email</a> if you have any questions.</p> </div> <div class="news"> <h2>news</h2> <div class="table-responsive" style="max-height: 10vw"> <table class="table table-sm table-borderless"> <tr> <th scope="row">Feb 10, 2024</th> <td> Finally, the <a href="https://doi.org/10.1186/s13075-024-03280-2" rel="external nofollow noopener" target="_blank">Large-scale integrative analysis of juvenile idiopathic arthritis for new insight into its pathogenesis</a> have been published in <a href="https://arthritis-research.biomedcentral.com/" rel="external nofollow noopener" target="_blank">Arthritis Research &amp; Therapy</a>! <img class="emoji" title=":fire:" alt=":fire:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f525.png" height="20" width="20"> </td> </tr> <tr> <th scope="row">Dec 22, 2023</th> <td> WOWOW!!! The <a href="https://www.nature.com/nrneurol/" rel="external nofollow noopener" target="_blank">Nature Reviews Neurology</a> features our <a href="https://academic.oup.com/hmg/advance-article/doi/10.1093/hmg/ddad174/7308735" rel="external nofollow noopener" target="_blank">stroke multiome-wide association study</a> paper! Please check <a href="https://www.nature.com/articles/s41582-023-00908-w" rel="external nofollow noopener" target="_blank">News &amp; Views</a> <a href="assets/pdf/NatRevNeu_stroke_twas.pdf">[pdf]</a> in Nature Reviews Neurology<img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> </td> </tr> <tr> <th scope="row">Oct 13, 2023</th> <td> The <a href="https://academic.oup.com/hmg/advance-article/doi/10.1093/hmg/ddad174/7308735" rel="external nofollow noopener" target="_blank">Novel insight into the etiology of ischemic stroke gained by integrative multiome-wide association study</a> has been accepted for publication in <a href="https://academic.oup.com/hmg" rel="external nofollow noopener" target="_blank">Human Molecular Genetics</a>! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> </td> </tr> <tr> <th scope="row">Jul 10, 2023</th> <td> I have recently become a member of the <a href="https://www.cedars-sinai.edu/research/departments-institutes/computational-biomedicine.html" rel="external nofollow noopener" target="_blank">Department of Computational Biomedicine</a> at <a href="https://www.cedars-sinai.org/" rel="external nofollow noopener" target="_blank">Cedars-Sinai Medical Center</a>!! </td> </tr> <tr> <th scope="row">Apr 30, 2021</th> <td> Finally, the <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02353-8" rel="external nofollow noopener" target="_blank">MARS: leveraging allelic heterogeneity to increase power of association testing</a> has been published! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> </td> </tr> </table> </div> </div> <div class="publications"> <h2>selected publications</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Analysis (Corresponding)</abbr></div> <div id="jung2023novel" class="col-sm-8"> <div class="title">Novel insight into the etiology of ischemic stroke gained by integrative multiome-wide association study</div> <div class="author"> Junghyun Jung*, Zeyun Lu, Adam Smith, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Nicholas Mancuso*' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Human Molecular Genetics</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://academic.oup.com/hmg/advance-article/doi/10.1093/hmg/ddad174/7308735" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/twas_stroke.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Stroke, characterized by sudden neurological deficits, is the second leading cause of death worldwide. Although genome-wide association studies (GWAS) have successfully identified many genomic regions associated with ischemic stroke (IS), the genes underlying risk and their regulatory mechanisms remain elusive. Here, we integrate a large-scale GWAS (N = 1 296 908) for IS together with molecular QTLs data, including mRNA, splicing, enhancer RNA (eRNA), and protein expression data from up to 50 tissues (total N = 11 588). We identify 136 genes/eRNA/proteins associated with IS risk across 60 independent genomic regions and find IS risk is most enriched for eQTLs in arterial and brain-related tissues. Focusing on IS-relevant tissues, we prioritize 9 genes/proteins using probabilistic fine-mapping TWAS analyses. In addition, we discover that blood cell traits, particularly reticulocyte cells, have shared genetic contributions with IS using TWAS-based pheWAS and genetic correlation analysis. Lastly, we integrate our findings with a large-scale pharmacological database and identify a secondary bile acid, deoxycholic acid, as a potential therapeutic component. Our work highlights IS risk genes/splicing-sites/enhancer activity/proteins with their phenotypic consequences using relevant tissues as well as identify potential therapeutic candidates for IS.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Method (First)</abbr></div> <div id="mars" class="col-sm-8"> <div class="title">MARS: leveraging allelic heterogeneity to increase power of association testing</div> <div class="author"> Farhad Hormozdiari†, Junghyun Jung†, Eleazar Eskin, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jong Wha J Joo' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Genome biology</em> Apr 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02353-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-021-02353-8.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>In standard genome-wide association studies (GWAS), the standard association test is underpowered to detect associations between loci with multiple causal variants with small effect sizes. We propose a statistical method, Model-based Association test Reflecting causal Status (MARS), that finds associations between variants in risk loci and a phenotype, considering the causal status of variants, only requiring the existing summary statistics to detect associated risk loci. Utilizing extensive simulated data and real data, we show that MARS increases the power of detecting true associated risk loci compared to previous approaches that consider multiple variants, while controlling the type I error.</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%6A%75%6E%67%68%79%75%6E.%6A%75%6E%67@%63%73%68%73.%6F%72%67" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0001-6832-4368" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=3Va15VcAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/junghyunJJ" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/junghyun-jung-0197b29b" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a> <a href="https://twitter.com/junghyunjj219" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a> </div> <div class="contact-note"> Please feel free to contact me through email if you have any questions. </div> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2025 Junghyun Jung. Last updated: June 23, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>